src/CallFire/Api/DocumentedClient.php

```diff
@@ -0,0 +1,1819 @@
+<?php
```

Comment:
Do we want to commit a pre-generated version of this? It'll become outdated fairly quickly, or we'll have to update it regularly.